src/highlevel_feature_extractor.cpp

```diff
@@ -55,10 +55,9 @@ const std::vector<float>& HighLevelFeatureExtractor::ExtractFeatures(
   // Features about the ball
   Vector2D ball_pos = wm.ball().pos();
   angleDistToPoint(self_pos, ball_pos, th, r);
-  // Feature[3]: Dist to ball
-  addNormFeature(r, 0, maxR);
-  // Feature[4]: Ang to ball
-  addNormFeature(th, -M_PI, M_PI);
+  // Feature[3] and [4]: (x,y) postition of the ball
+  addNormFeature(ball_pos.x, -SP.pitchHalfLength()-tolerance_x, tolerance_x);
```

Comment:
This needs to change depending on whether the agent is playing offense or defense. See code for computing X position:

// Feature[0]: X-postion
  if (playingOffense) {
    addNormFeature(self_pos.x, -tolerance_x, SP.pitchHalfLength() + tolerance_x);
  } else {
    addNormFeature(self_pos.x, -SP.pitchHalfLength()-tolerance_x, tolerance_x);
  }